src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -141,6 +141,18 @@
                                             </div>
                                             {% if message.htmlBody is defined %}
                                                 {# Email instance #}
+                                                <div class="tab">
+                                                    <h3 class="tab-title">HTML preview</h3>
+                                                    <div class="tab-content">
+                                                        <pre class="prewrap" style="max-height: 600px">
```

Comment:
why using a `pre` tag as a wrapper of the iframe ?

Reply:
To force the mail "area"
But I might be removed, I did not test it.